README.md

```diff
@@ -1,2 +1,191 @@
-# Financial-Eye
-Financial Eye is a free and easy-to-use tool that can help you to stay informed about the financial health of your investments.
+<a name="readme-top"></a>
+
+<!--
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+-->
+
+<div align="center">
+  <h1><b>Financial Eye</b></h1>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Financial Eye <a name="about-project"></a>
+
+**Financial Eye** Financial Eye is a web app that provides users with access to financial statements for any stock. The app is easy to use and allows users to search for stocks by ticker symbol.Once a stock has been selected, users can view the company's financial statements, including the balance sheet, income statement, and cash flow statement.
+
+## 🛠 Built With <a name="built-with"></a>
+
+- HTML
+- CSS
+- JavaScript
+- React
+- Redux
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/es/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/es/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://developer.mozilla.org/es/docs/Web/JavaScript">JavaScript</a></li>
+  </ul>
+</details>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Key Features <a name="key-features"></a>
+- In the home page there is a list with the biggest 10 companies in the American stock market.
+- When the user clicks on any of these companies it will move him to a details page where
+he can select which finanial statment to preview.
+- There is an interface called search when the iuser clicks on the search link it will move 
+him to a page where he can enter the ticker and the financial statement to display. 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> Here is a link to the demo project.
+
+- [Live Demo Link](https://finacial-eye.onrender.com/)
+- [presenrtation](https://drive.google.com/file/d/1COt0HTIfBTL00sWg9NqQlubfSywghIBQ/view?usp=sharing)
```

Comment:
*  According to the project requirements, the maximum length of the video should be 5 minutes and nothing more. Your current video is 6 minutes and 38 seconds which passes the limit. Please adjust your video presentation to fit within the required time limit [check requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md#:~:text=Once%20you%20have,a%20small%20picture.)